predicators/ground_truth_models/doors/nsrts.py

```diff
@@ -171,3 +171,131 @@ def open_door_sampler(state: State, goal: Set[GroundAtom],
         nsrts.add(move_through_door_nsrt)
 
         return nsrts
+
+
+class DoorknobsGroundTruthNSRTFactory(DoorsGroundTruthNSRTFactory):
```

Comment:
looks like there's substantial code duplication here as well; I think there should be a way to merge/simplify things: see the nsrts.py file for the cover env for an example of making one NSRT factory that handles a number of different variants of the same env